Automattic/wp-calypso
# Jetpack Manage: Pricing page - styling adjustments to match design specifications

Related to https://github.com/Automattic/jetpack-manage/issues/266

## Proposed Changes

* This PR includes several style related changes, as well as minor location adjustments. Specifically:
* All font sizes should match design specs
* Variant selection is above the price in both the VaultPress Backup and the Security cards.
* Prices will show 'US$' in front.
* FAQ section removed to match design specs
* Backup, Boost and Akismet product icons match currently existing icons, not design specs, as discussed here: pf36In-Gb-p2#comment-1243 
* The “per bundle per month” and “per license per month” text that shows for bundles greater than 1 will now show on a new line. See pf36In-Gb-p2#comment-1297 

Design specs are at pf36In-Gb-p2.


## Testing Instructions

* To test, check out this branch and open `http://jetpack.cloud.localhost:3000/manage/pricing`. 
* Make sure the above listed design changes (plus everything else on the page) matches the design specs
* Note that this PR is just to make design / layout tweaks on the `/manage/pricing` page itself, so doesn't consider any functionality issues such as where the buttons lead to as some issues are still being worked out there.


Screenshot showing the full page:

<img width="336" alt="Screenshot 2024-02-02 at 09 44 49" src="https://github.com/Automattic/wp-calypso/assets/16754605/04d240e8-9b1b-4777-bfa0-b9eb2295368a">


## Pre-merge Checklist



- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?lat
